descriptors/ModuleDescriptor-template.json

```diff
@@ -17,6 +17,39 @@
           ],
           "modulePermissions": [
           ]
+        },
+        {
+          "methods": [
+            "GET"
+          ],
+          "pathPattern": "/data-export/fileDefinitions/{fileDefinitionId}",
+          "permissionsRequired": [
+            "data-export.filedefinitions.get"
+          ],
+          "modulePermissions": [
+          ]
+        },
+        {
+          "methods": [
+            "POST"
+          ],
+          "pathPattern": "/data-export/fileDefinitions/{fileDefinitionId}",
+          "permissionsRequired": [
+            "data-export.filedefinitions.uploadfile"
+          ],
+          "modulePermissions": [
+          ]
+        },
+        {
+          "methods": [
+            "POST"
+          ],
+          "pathPattern": "/data-export/fileDefinitions",
+          "permissionsRequired": [
```

Comment:
Also, it would be good to create a permission set containing all interface permissions, so that UI can use that